src/_grid.scss

```diff
@@ -316,6 +362,7 @@
  */
 
 .l-grid.l-grid--gutterless {
+    margin-bottom: 0;
```

Comment:
Do we need to reset this? The base case doesn't have a `margin-bottom` property anyway